SPHINXsys/src/shared/bodies/body_relation.cpp

```diff
@@ -166,6 +169,16 @@ namespace SPH
 		}
 	}
 	//=================================================================================================//
+	void SolidContactBodyRelation::initialization()
+	{
+		for (size_t k = 0; k != contact_bodies_.size(); ++k)
+		{
+			target_mesh_cell_linked_lists_.push_back(dynamic_cast<MeshCellLinkedList*>(contact_bodies_[k]->mesh_cell_linked_list_));
+			get_search_ranges_.push_back(new SearchRangeMultiResolution(sph_body_, contact_bodies_[k]));
```

Comment:
where the pointers are deleted? consider to use unique pointer?

Reply:
These are one shot pointers. I will check all pointers in the code in a specialized branch.